suite-native/assets/src/assetsSelectors.ts

```diff
@@ -69,10 +73,12 @@ export const selectDeviceAssetsWithBalances = (state: AssetsRootState & DeviceRo
             shouldIncludeStaking: true,
         });
 
+        const cryptoBalanceWithStaking = getAccountCryptoBalanceWithStakingFormatted(account);
```

Comment:
`getAccountFiatBalance` few lines before should be capable including staking balance and I think it's probably happening by accident because there is `shouldIncludeStaking: true`

Reply:
I probably got it wrong, but getAccountFiatBalance calculated correctly. The issue occurs for the asset's crypto balance (account.formattedBalance). It's calculated without staking, as I see.